rust/migration/src/m20220101_000002_create_table.rs

```diff
@@ -84,7 +84,7 @@ impl MigrationTrait for Migration {
                             .default(SimpleExpr::FunctionCall(PgFunc::gen_random_uuid())),
                     )
                     .col(ColumnDef::new(VisibleFile::Path).string().not_null())
-                    .col(ColumnDef::new(VisibleFile::Hash).ezblob())
+                    .col(ColumnDef::new(VisibleFile::Hash).string())
```

Comment:
Do we think all `Hash` types should become strings? I noticed `Job::Hash` above is still using `ezblob`

Reply:
I believe `Job::Hash` is never exposed via the API so using `ezblob` is allowed. That said probably still makes more sense to have them be uniform